tools/importer-rest-api-specs/components/parser/models.go

```diff
@@ -451,7 +451,13 @@ func (d *SwaggerDefinition) findAncestorType(input spec.Schema) (*string, *strin
 	return nil, nil, nil
 }
 
-func (d SwaggerDefinition) parseObjectDefinition(modelName, propertyName string, input *spec.Schema, known internal.ParseResult) (*models.ObjectDefinition, *internal.ParseResult, error) {
+// if `inputForModel` is false, it means the `input` schema cannot be used to parse the model of `modelName`
+func (d SwaggerDefinition) parseObjectDefinition(
+	modelName, propertyName string,
+	input *spec.Schema,
+	known internal.ParseResult,
+	isInputForModel bool,
```

Comment:
since this is being used to determine if we're passing a model, or pulling inlined information from a field - this is probably better named as:

```suggestion
	parsingModel bool,
```